packages/inline-video/style.module.css

```diff
@@ -0,0 +1,79 @@
+.inlineVideo {
+  --gradient-size: 32px;
+}
+
+.videoContainer {
+  --gradient-stops: #0654ef 0%, #cde6fe 100%;
+
+  width: 100%;
+  position: relative;
+
+  @media (--medium-up) {
+    @nest .left & {
+      padding-left: var(--gradient-size);
+    }
+
+    @nest .right & {
+      padding-right: var(--gradient-size);
+    }
+
+    &::before {
+      position: absolute;
+      content: '';
+      width: 20%;
+      height: calc(100% - var(--gradient-size) * 2);
+      top: var(--gradient-size);
+      border-radius: 8px;
+      z-index: -1;
+      background-image: linear-gradient(0deg, var(--gradient-stops));
+
+      @nest .left & {
+        left: 0;
+      }
+
+      @nest .right & {
+        right: 0;
+      }
+    }
+  }
+
+  &.infrastructure {
+    --gradient-stops: #8763d8 0%, #8cddff 100%;
+  }
+
+  &.security {
+    --gradient-stops: #f26c64 0%, #ffe68d 100%;
+  }
+
+  &.networking {
+    --gradient-stops: #c750b7 0%, #ffb3b7 100%;
+  }
+
+  &.applications {
+    --gradient-stops: #5493e4 0%, #5bd0d9 50%, #a3f7d7 100%;
+  }
```

Comment:
these are available via wpl tokens https://github.com/hashicorp/mktg-global-styles/blob/main/custom-properties/color.css#L132-L191

Reply:
@alexcarpenter Hmm...I've replaced all the values here with their proper tokens, and yet the actual declaration doesn't apply the value initially assigned to `--gradient`? Changing the `solution` prop to a supported value _does_ allow the correct gradient to show, however. 🤔 

Maybe I'm running into some kind of scoping or syntax issue?

Reply:
Update: Just needed to update deps! We're good to go as of the latest commit:
https://github.com/hashicorp/react-components/pull/789/commits/65a6e3c9295c7a3fed292c039321bc5ae3469648